pages/common/rustc.md

```diff
@@ -0,0 +1,19 @@
+# rustc
+
+> The Rust compiler.
+
+- Compile a single file:
+
+`rustc {{file.rs}}`
+
+- Compile with high optimization:
+
+`rustc -O {{file.rs}}`
+
+- Compile with debugging information:
+
+`rustc -g {{file.rs}}`
```

Comment:
No, I don't think the extra complexity is worth it. I suppose one can hang on to the "g" in "debug" for a mnemonic, which should still be more memorable than `-C debuginfo=2`. Thanks for the explanation.